Jenkinsfile

```
@@ -35,44 +39,45 @@ def runBrowserTests(name, browser, os, bucket, buckets, runAll) {
   runTests(name, bedrockCommand, runAll);
 }
 
-def runPhantomTests(runAll) {
-  def bedrockCommand = "yarn grunt phantomjs-auto";
-  runTests("PhantomJS", bedrockCommand, runAll);
+def runHeadlessTests(Boolean runAll) {
+  def bedrockCommand = "yarn grunt headless-auto";
+  runTests("chrome-headless", bedrockCommand, runAll);
 }
 
-standardProperties()
-
 def gitMerge(String primaryBranch) {
-  if (BRANCH_NAME != primaryBranch) {
+  if (env.BRANCH_NAME != primaryBranch) {
     echo "Merging ${primaryBranch} into this branch to run tests"
     exec("git merge --no-commit --no-ff origin/${primaryBranch}")
   }
 }
 
-node("primary") {
-  timestamps {
-    checkout scm
-
-    def props = readProperties file: 'build.properties'
-
-    def primaryBranch = props.primaryBranch
+timestamps {
+  // TinyMCE builds need more CPU and RAM (especially eslint)
+  // NOTE: Ensure not to go over 7.5 CPU/RAM due to EC2 node sizes and the jnlp container requirements
+  tinyPods.node([
```

On `develop` and `master` we are using `tinyPods.nodeBrowser`. Just wanted to check if what we have here is fine
```suggestion
  ttinyPods.nodeBrowser([
```

---

@mkzhx this should run a selenium chrome container to run the headless tests. Recommended

---

Thanks guys, suggestion applied.